lib/vagrant-digitalocean/actions/rebuild.rb

```diff
@@ -15,10 +15,14 @@ def initialize(app, env)
         end
 
         def call(env)
+          search = @machine.provider_config.image_name.nil? ?
+            {:slug => @machine.provider_config.image} :
+            {:name => @machine.provider_config.image_name}
```

Comment:
So `image_name` supersedes `image`?  That could be a bit confusing if both are specified but for different images.